web/.eslintrc.json

```diff
@@ -24,12 +20,7 @@
 
     "react/forbid-prop-types": ["error", { "forbid": ["any"] }],
     "react/jsx-filename-extension": 0,
-    "react/no-array-index-key": "warn",
-
-    // unnecessary due to prettier
-    "function-paren-newline": "off",
-    "indent": "off",
-    "object-curly-newline": "off"
+    "react/no-array-index-key": "warn"
```

Comment:
We should discuss this particular rule elsewhere sometime.

Reply:
agreed, would love to kick this